packages/common/src/authentication/AuthContext.tsx

```diff
@@ -15,6 +15,7 @@ type User = {
 };
 
 interface Store {
+  __typename: 'StoreNode';
```

Comment:
as to use `StoreRowFragment` generated type instead of the `Store` types.. 🤔  

Bad idea?

Reply:
I did wonder about doing that. didn't like having the separate types